index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Emmanuel Hagan" />
+  <title>Hagan's Portifio</title>
+    <!-- Stylesheet -->
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+  <!--  Header  -->
+  <header class="toolbar-container">
+      <a href="#" class="logo-heading" > Hagan</a>
+   <!---  
+      <nav class="hamburger-menu">
+       <ul class="header-nav-links">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li> 
+        </ul>
+      </nav>
+    -->
+      <img class="icon" src="assets/image/icon-menu.png"  alt="Icon menu">
+  </header>
+  <!-- End Header  -->
+
+  <!-- Section  -->
+  <section class="headline">
+    <div class="headline-content-wrapper">
+      <h1 class="intro">I’m Emmanuel Hagan Glad to see you!</h1>
+    </div>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      
+        <a href="" class="connect">
+          <h1>LET’S CONNECT</h1>
+        </a>
```

Comment:
- [ ] There seems to be too much space on the left side of the logo link. please try to match the Figma design as closely as possible.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/164813965-f6d62c98-bf35-473a-bc15-fba87a70c9b1.png)|![image](https://user-images.githubusercontent.com/18730106/164813978-94e37f56-ed76-481b-9dda-de1ec9a8141d.png)